app/components/channel_list/categories/body/channel/channel_list_item.tsx

```diff
@@ -61,28 +61,30 @@ const textStyle = StyleSheet.create({
 });
 
 type Props = {
-    channel: Pick<ChannelModel, 'deleteAt' | 'displayName' | 'name' | 'shared' | 'type'>;
+    channel: ChannelModel;
     isActive: boolean;
-    isOwnDirectMessage: boolean;
     isMuted: boolean;
     myChannel?: MyChannelModel;
     collapsed: boolean;
+    currentUserId: string;
 }
 
-const ChannelListItem = ({channel, isActive, isOwnDirectMessage, isMuted, myChannel, collapsed}: Props) => {
+const ChannelListItem = ({channel, isActive, currentUserId, isMuted, myChannel, collapsed}: Props) => {
     const {formatMessage} = useIntl();
     const theme = useTheme();
     const styles = getStyleSheet(theme);
     const serverUrl = useServerUrl();
 
+    const isOwnDirectMessage = (channel.type === General.DM_CHANNEL) && currentUserId === getUserIdFromChannelName(currentUserId, channel.name);
+
     // Make it brighter if it's not muted, and highlighted or has unreads
     const bright = !isMuted && (isActive || (myChannel && (myChannel.isUnread || myChannel.mentionsCount > 0)));
 
     const sharedValue = useSharedValue(collapsed && !bright);
 
     useEffect(() => {
         sharedValue.value = collapsed && !bright;
-    }, [collapsed, bright]);
+    }, [collapsed && !bright]);
```

Comment:
We saw some performance degradation on the Animated Style when it was bright. We deduced that it could only be because we are reassigning a value to `sharedValue` here. Changing this removed the degradation, so we have deduced that assigning a value to a shared value, even if the value is the same as it was before, triggers some processing that we should better not trigger.